docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -12,14 +12,11 @@ Depending on your role on the Git provider you will have different permissions o
       <th>Provider</th>
       <th>Role</th>
       <th>Join organization</th>
-      <th>Ignore issues and files</th>
-      <th>Add repository</th>
+      <th>View private repository</th>
```

Comment:
@IGFCoimbra I changed the title of this column to clarify that it applies to private repositories only.

The fact that public repositories can be viewed by anyone on Codacy is not explicit here, but if we find that this is important to be explained somewhere we can add that detail as well.